src/AvifLoad.cs

```diff
@@ -297,8 +296,11 @@ static IBitmapSource<ColorPbgra32> PQToDisplayP3(
                             srcColorContext,
                             dstColorContext,
                             alphaMode);
+                        UnPremultiplyEffect2 unPremultiplyEffect = new UnPremultiplyEffect2(dc);
+                        unPremultiplyEffect.SetValueByName("Enabled", alphaMode == ColorManagementAlphaMode.Premultiplied);
```

Comment:
`unPremultiplyEffect.Properties.Enabled.Set(value);`

Reply:
In fact you can also just use the other constructor,
```cs
UnPremultiplyEffect2 unPremultiplyEffect = new UnPremultiplyEffect(dc, colorMgmtEffect, trueOrFalse);
```

Reply:
I updated the code to use Enabled.SetValue, but I did not see that constructor overload.